BREAKING.md

```diff
@@ -134,8 +134,12 @@ The following members of the `@fluidframework/driver-utils` package have been de
 The following members of the `@fluidframework/aqueduct` package have been deprecated and will be removed in an upcoming release:
 
 -   `waitForAttach()`
+
```

Comment:
This looks wrong formatting - should be no blank lines between any of these bullet points